docs/core/whats-new/dotnet-10/overview.md

```diff
@@ -23,14 +23,14 @@ For more information, see [What's new in the .NET 10 runtime](runtime.md).
 
 ## .NET libraries
 
-The .NET 10 libraries introduce new APIs in cryptography, globalization, numerics, serialization, collections, and diagnostics, and when working with ZIP files. New JSON serialization options include disallowing duplicate properties and strict serialization settings. Post-quantum cryptography support has been expanded with Windows Cryptography API: Next Generation (CNG) support.
+The .NET 10 libraries introduce new APIs in cryptography, globalization, numerics, serialization, collections, and diagnostics, and when working with ZIP files. New JSON serialization options include disallowing duplicate properties and strict serialization settings. Post-quantum cryptography support has been expanded with Windows Cryptography API: Next Generation (CNG) support. Preview 7 adds Windows process group support, AES KeyWrap with Padding, enhanced post-quantum cryptography with ML-DSA updates, PipeReader support for JSON serialization, WebSocketStream for simplified WebSocket usage, and TLS 1.3 support for macOS clients.
```

Comment:
Incorporate the new preview 7 features into the flow of the paragraph instead of calling them out separately. For example, PipeReader support for JSON serialization should be with other JSON additions, the new cryptography options should be included with the "new APIs in cryptography" sentence. The first paragraph says this file was updated for Preview 7. No need to repeat that point throughout the article.